### PR TITLE
ENT-10250: Now mentiones lack of square bracket glob expression support on Windows

### PR DESCRIPTION
**Pull request by larsewi**

### CODE REVIEW

File: reference/functions/findfiles.markdown [hunk 1]
```diff
@@ -17,14 +17,15 @@ match all files in one directory that end in `.cf` but it won't search
 across directories.  `*/*.cf` on the other hand will look two levels
 deep.
 * `?` matches a single letter
-* `[a-z]` matches any letter from `a` to `z`
+* `[a-z]` matches any letter from `a` to `z` (not yet supported on Windows)
```
Comment (nickanderson) [inline reference/functions/findfiles.markdown:20]: @larsewi do we have a ticket for adding this support?
- It should link to the docs here to update them when support is added.

**Review by nickanderson — APPROVED** [review]

File: reference/functions/findfiles.markdown (hunk 1 above)
Comment (larsewi) [reply reference/functions/findfiles.markdown:20]: https://northerntech.atlassian.net/browse/CFE-4255

Comment (nickanderson) [reply reference/functions/findfiles.markdown:20]: ty

**Comment by nickanderson** [comment]
Picked to 3.21 and 3.18

**Pull request merged**